### ISSUE TITLE
Test failure JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.dll

### ISSUE THREAD

**Issue by v-wenyuxu** [dotnet/runtime#88882]
**Failed in:** [runtime-coreclr jitstress 20230712.1](https://dev.azure.com/dnceng-public/public/_build/results?buildId=338051&view=ms.vss-test-web.build-test-results-tab&runId=7034114&resultId=118892&paneView=debug)

**Failed tests:**
```
coreclr windows x64 Checked jitstress2 @ Windows.10.Amd64.Open
    - JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.dll
coreclr windows x64 Checked jitstress2 @ Windows.10.Amd64.Open
    - JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-5/regex-redux-5.dll
```

**Error message:**
```
Failed to compare two elements in the array.
```
**Stack trace:**
```
   at System.Collections.Generic.ArraySortHelper`1.Sort(Span`1 keys, Comparison`1 comparer)
   at System.Text.RegularExpressions.RegexFindOptimizations.FindFixedDistanceString(List`1 fixedDistanceSets) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFindOptimizations.cs:line 272
   at System.Text.RegularExpressions.RegexFindOptimizations..ctor(RegexNode root, RegexOptions options) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFindOptimizations.cs:line 147
   at System.Text.RegularExpressions.RegexTree..ctor(RegexNode root, Int32 captureCount, String[] captureNames, Hashtable captureNameToNumberMapping, Hashtable captureNumberSparseMapping, RegexOptions options, CultureInfo culture) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexTree.cs:line 80
   at System.Text.RegularExpressions.RegexParser.Parse(String pattern, RegexOptions options, CultureInfo culture) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs:line 131
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, CultureInfo culture) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs:line 96
   at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options) in /_/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs:line 61
   at BenchmarksGame.RegexRedux_1.Bench(TextReader inputReader, Boolean verbose)
   at BenchmarksGame.RegexRedux_1.TestEntryPoint()
   at Program.<<Main>$>g__TestExecutor74|0_75(StreamWriter tempLogSw, StreamWriter statsCsvSw, <>c__DisplayClass0_0&)
```

Queued | Pipeline | Pipeline_Configuration | OS | Arch | Test | Outcome | Properties | BuildReason
-- | -- | -- | -- | -- | -- | -- | -- | --
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
<details>
<summary>More failures</summary>

Queued | Pipeline | Pipeline_Configuration | OS | Arch | Test | Outcome | Properties | BuildReason
-- | -- | -- | -- | -- | -- | -- | -- | --
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-05-01T21:56:38.34Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildNumber":"20230501.4","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests_Interp","System.PhaseAttempt":"1","architecture":"x64","AzurePipelinesTestRunId":"5052790","System.StageAttempt":"1","DefinitionName":"runtime-extra-platforms","System.JobAttempt":"1","System.StageName":"Build","Project":"public","Reason":"Schedule","DefinitionId":"154","BuildId":"258590","System.JobName":"__default","System.JobId":"e847dbc9-1eac-5acf-c55e-fa567d2d051e","configuration":"Release","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open"} | Schedule
2023-04-26T06:10:17.798Z | runtime-coreclr jitstress | Checked-jitstress2 | osx.1200.amd64.open | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildId":"252731","Reason":"Schedule","System.PhaseName":"run_test_p1__osx_x64_checked","System.PhaseAttempt":"1","DefinitionName":"runtime-coreclr jitstress","AzurePipelinesTestRunId":"4906790","Project":"public","operatingSystem":"OSX.1200.Amd64.Open","System.StageAttempt":"1","CollectionUri":"https://dev.azure.com/dnceng-public/","configuration":"Checked-jitstress2","DefinitionId":"109","System.StageName":"Build","BuildNumber":"20230425.5","System.JobId":"bf644a0b-2dbe-5a03-d8c6-a86fdd97c823","architecture":"x64","System.JobName":"__default","System.JobAttempt":"1"} | Schedule
2023-04-26T05:25:12.735Z | runtime-coreclr jitstress | Checked-jitstress2 | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.StageAttempt":"1","CollectionUri":"https://dev.azure.com/dnceng-public/","operatingSystem":"Ubuntu.1804.Amd64.Open","BuildId":"252731","System.JobAttempt":"1","Project":"public","DefinitionId":"109","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","configuration":"Checked-jitstress2","System.JobName":"__default","Reason":"Schedule","DefinitionName":"runtime-coreclr jitstress","System.StageName":"Build","BuildNumber":"20230425.5","architecture":"x64","AzurePipelinesTestRunId":"4906396","System.PhaseAttempt":"1","System.PhaseName":"run_test_p1__linux_x64_checked"} | Schedule
2023-04-26T05:25:10.422Z | runtime-coreclr jitstress | Checked-jitstress1 | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"configuration":"Checked-jitstress1","BuildId":"252731","System.JobAttempt":"1","System.PhaseName":"run_test_p1__linux_x64_checked","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","AzurePipelinesTestRunId":"4906398","CollectionUri":"https://dev.azure.com/dnceng-public/","System.StageName":"Build","operatingSystem":"Ubuntu.1804.Amd64.Open","BuildNumber":"20230425.5","System.StageAttempt":"1","System.JobName":"__default","System.PhaseAttempt":"1","DefinitionId":"109","architecture":"x64","Reason":"Schedule","Project":"public","DefinitionName":"runtime-coreclr jitstress"} | Schedule
2023-04-15T21:34:40.756Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_2_x86_nosse | osx.1200.amd64.open | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"operatingSystem":"OSX.1200.Amd64.Open","Reason":"Schedule","BuildId":"240798","BuildNumber":"20230415.1","configuration":"Checked-jitstress_isas_2_x86_nosse","DefinitionId":"115","architecture":"x64","System.StageName":"Build","System.PhaseName":"run_test_p1__osx_x64_checked","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.JobAttempt":"1","System.JobName":"__default","Project":"public","System.JobId":"bf644a0b-2dbe-5a03-d8c6-a86fdd97c823","System.StageAttempt":"1","AzurePipelinesTestRunId":"4624132","CollectionUri":"https://dev.azure.com/dnceng-public/","System.PhaseAttempt":"1"} | Schedule
2023-04-15T21:34:38.305Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_2_x86_nohwintrinsic | osx.1200.amd64.open | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.JobAttempt":"1","Reason":"Schedule","BuildId":"240798","System.PhaseName":"run_test_p1__osx_x64_checked","System.JobId":"bf644a0b-2dbe-5a03-d8c6-a86fdd97c823","Project":"public","BuildNumber":"20230415.1","AzurePipelinesTestRunId":"4624144","operatingSystem":"OSX.1200.Amd64.Open","System.JobName":"__default","DefinitionId":"115","System.StageName":"Build","DefinitionName":"runtime-coreclr jitstress-isas-x86","CollectionUri":"https://dev.azure.com/dnceng-public/","System.StageAttempt":"1","System.PhaseAttempt":"1","configuration":"Checked-jitstress_isas_2_x86_nohwintrinsic","architecture":"x64"} | Schedule
2023-04-15T21:07:25.828Z | runtime-coreclr jitstress-isas-arm | Checked-jitstress_isas_nohwintrinsic | osx.1200.arm64.open | arm64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.JobAttempt":"1","architecture":"arm64","Reason":"Schedule","System.StageName":"Build","CollectionUri":"https://dev.azure.com/dnceng-public/","System.PhaseAttempt":"1","BuildNumber":"20230415.1","operatingSystem":"OSX.1200.ARM64.Open","System.JobName":"__default","System.StageAttempt":"1","System.PhaseName":"run_test_p1__osx_arm64_checked","BuildId":"240790","AzurePipelinesTestRunId":"4623820","Project":"public","configuration":"Checked-jitstress_isas_nohwintrinsic","System.JobId":"765fd5e9-e47f-5cb0-4cbf-499eaaaee93e","DefinitionId":"116","DefinitionName":"runtime-coreclr jitstress-isas-arm"} | Schedule
2023-04-15T20:54:26.022Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_1_x86_nohwintrinsic | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"CollectionUri":"https://dev.azure.com/dnceng-public/","System.PhaseName":"run_test_p1__linux_x64_checked","Project":"public","System.JobName":"__default","DefinitionName":"runtime-coreclr jitstress-isas-x86","DefinitionId":"115","BuildNumber":"20230415.1","AzurePipelinesTestRunId":"4623586","System.StageAttempt":"1","BuildId":"240798","System.PhaseAttempt":"1","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","configuration":"Checked-jitstress_isas_1_x86_nohwintrinsic","System.StageName":"Build","System.JobAttempt":"1","operatingSystem":"Ubuntu.1804.Amd64.Open","architecture":"x64","Reason":"Schedule"} | Schedule
2023-04-15T20:54:21.611Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_2_x86_nohwintrinsic | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"DefinitionId":"115","Project":"public","System.PhaseAttempt":"1","System.JobName":"__default","CollectionUri":"https://dev.azure.com/dnceng-public/","BuildNumber":"20230415.1","AzurePipelinesTestRunId":"4623580","BuildId":"240798","Reason":"Schedule","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","architecture":"x64","configuration":"Checked-jitstress_isas_2_x86_nohwintrinsic","System.JobAttempt":"1","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.StageName":"Build","operatingSystem":"Ubuntu.1804.Amd64.Open","System.PhaseName":"run_test_p1__linux_x64_checked","System.StageAttempt":"1"} | Schedule
2023-04-15T20:54:03.001Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_1_x86_nosse | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"CollectionUri":"https://dev.azure.com/dnceng-public/","configuration":"Checked-jitstress_isas_1_x86_nosse","System.JobAttempt":"1","DefinitionId":"115","System.PhaseName":"run_test_p1__linux_x64_checked","Reason":"Schedule","architecture":"x64","Project":"public","BuildId":"240798","AzurePipelinesTestRunId":"4623562","BuildNumber":"20230415.1","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","System.StageAttempt":"1","System.StageName":"Build","operatingSystem":"Ubuntu.1804.Amd64.Open","System.JobName":"__default","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.PhaseAttempt":"1"} | Schedule
2023-04-15T20:54:00.679Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_x86_nosse2 | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.PhaseName":"run_test_p1__linux_x64_checked","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","BuildId":"240798","AzurePipelinesTestRunId":"4623648","DefinitionId":"115","System.StageAttempt":"1","CollectionUri":"https://dev.azure.com/dnceng-public/","System.JobName":"__default","Reason":"Schedule","System.StageName":"Build","BuildNumber":"20230415.1","System.JobAttempt":"1","operatingSystem":"Ubuntu.1804.Amd64.Open","Project":"public","DefinitionName":"runtime-coreclr jitstress-isas-x86","configuration":"Checked-jitstress_isas_x86_nosse2","architecture":"x64","System.PhaseAttempt":"1"} | Schedule
2023-04-15T20:53:42.274Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_2_x86_nosse | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"architecture":"x64","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","System.JobName":"__default","DefinitionName":"runtime-coreclr jitstress-isas-x86","BuildNumber":"20230415.1","AzurePipelinesTestRunId":"4623634","Reason":"Schedule","configuration":"Checked-jitstress_isas_2_x86_nosse","BuildId":"240798","CollectionUri":"https://dev.azure.com/dnceng-public/","Project":"public","operatingSystem":"Ubuntu.1804.Amd64.Open","System.StageAttempt":"1","System.PhaseName":"run_test_p1__linux_x64_checked","System.StageName":"Build","DefinitionId":"115","System.PhaseAttempt":"1","System.JobAttempt":"1"} | Schedule
2023-04-15T20:53:41.065Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_1_x86_nosse2 | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildId":"240798","DefinitionId":"115","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","System.JobName":"__default","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.StageName":"Build","Reason":"Schedule","Project":"public","System.PhaseAttempt":"1","operatingSystem":"Ubuntu.1804.Amd64.Open","BuildNumber":"20230415.1","AzurePipelinesTestRunId":"4623610","System.StageAttempt":"1","configuration":"Checked-jitstress_isas_1_x86_nosse2","architecture":"x64","System.PhaseName":"run_test_p1__linux_x64_checked","CollectionUri":"https://dev.azure.com/dnceng-public/","System.JobAttempt":"1"} | Schedule
2023-04-15T20:53:38.67Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_x86_nosse | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","System.PhaseAttempt":"1","Project":"public","BuildId":"240798","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.PhaseName":"run_test_p1__linux_x64_checked","System.JobName":"__default","operatingSystem":"Ubuntu.1804.Amd64.Open","architecture":"x64","System.StageName":"Build","Reason":"Schedule","System.JobAttempt":"1","configuration":"Checked-jitstress_isas_x86_nosse","AzurePipelinesTestRunId":"4623638","DefinitionId":"115","System.StageAttempt":"1","CollectionUri":"https://dev.azure.com/dnceng-public/","BuildNumber":"20230415.1"} | Schedule
2023-04-15T20:53:37.507Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_x86_nohwintrinsic | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"AzurePipelinesTestRunId":"4623640","BuildNumber":"20230415.1","operatingSystem":"Ubuntu.1804.Amd64.Open","CollectionUri":"https://dev.azure.com/dnceng-public/","architecture":"x64","DefinitionName":"runtime-coreclr jitstress-isas-x86","Project":"public","System.StageAttempt":"1","System.PhaseName":"run_test_p1__linux_x64_checked","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","configuration":"Checked-jitstress_isas_x86_nohwintrinsic","System.JobAttempt":"1","System.JobName":"__default","System.StageName":"Build","System.PhaseAttempt":"1","DefinitionId":"115","Reason":"Schedule","BuildId":"240798"} | Schedule
2023-04-15T20:53:34.187Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_2_x86_nosse2 | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"Project":"public","configuration":"Checked-jitstress_isas_2_x86_nosse2","System.StageAttempt":"1","architecture":"x64","System.StageName":"Build","DefinitionName":"runtime-coreclr jitstress-isas-x86","System.PhaseName":"run_test_p1__linux_x64_checked","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","AzurePipelinesTestRunId":"4623592","operatingSystem":"Ubuntu.1804.Amd64.Open","CollectionUri":"https://dev.azure.com/dnceng-public/","System.PhaseAttempt":"1","BuildNumber":"20230415.1","System.JobName":"__default","System.JobAttempt":"1","DefinitionId":"115","Reason":"Schedule","BuildId":"240798"} | Schedule
2023-04-15T20:53:28.442Z | runtime-coreclr jitstress-isas-x86 | Checked-jitstress_isas_nohwintrinsic | ubuntu.1804.amd64.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"System.PhaseAttempt":"1","System.JobAttempt":"1","Reason":"Schedule","BuildId":"240798","AzurePipelinesTestRunId":"4623572","configuration":"Checked-jitstress_isas_nohwintrinsic","DefinitionId":"115","System.StageName":"Build","BuildNumber":"20230415.1","CollectionUri":"https://dev.azure.com/dnceng-public/","System.PhaseName":"run_test_p1__linux_x64_checked","System.JobName":"__default","System.JobId":"6dfec640-5b96-5d95-2a5b-7c236ea2d5c5","System.StageAttempt":"1","DefinitionName":"runtime-coreclr jitstress-isas-x86","architecture":"x64","operatingSystem":"Ubuntu.1804.Amd64.Open","Project":"public"} | Schedule
2023-04-15T20:26:19.582Z | runtime-coreclr jitstress-isas-arm | Checked-jitstress_isas_nohwintrinsic | ubuntu.1804.armarch.open | arm64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"BuildId":"240790","System.PhaseAttempt":"1","System.StageAttempt":"1","System.JobAttempt":"1","DefinitionName":"runtime-coreclr jitstress-isas-arm","operatingSystem":"(Ubuntu.1804.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8","Reason":"Schedule","configuration":"Checked-jitstress_isas_nohwintrinsic","Project":"public","System.JobId":"34b2f53b-add1-5e74-7a39-5a11483297fc","DefinitionId":"116","AzurePipelinesTestRunId":"4623274","BuildNumber":"20230415.1","CollectionUri":"https://dev.azure.com/dnceng-public/","architecture":"arm64","System.StageName":"Build","System.PhaseName":"run_test_p1__linux_arm64_checked","System.JobName":"__default"} | Schedule
2023-03-27T09:52:55.492Z | runtime-extra-platforms | Release | ubuntu.1804.amd64.android.29.open.rt | x64 | JIT/Performance/CodeQuality/BenchmarksGame/regex-redux/regex-redux-1/regex-redux-1.sh | Failed | {"architecture":"x64","System.JobId":"353cbaf6-57aa-5b05-dc8e-ce12fdf7e230","Project":"public","Reason":"Schedule","System.JobAttempt":"1","CollectionUri":"https://dev.azure.com/dnceng-public/","System.JobName":"__default","DefinitionName":"runtime-extra-platforms","System.StageAttempt":"1","configuration":"Release","AzurePipelinesTestRunId":"4080262","System.PhaseAttempt":"1","System.StageName":"Build","operatingSystem":"Ubuntu.1804.Amd64.Android.29.Open","System.PhaseName":"build_android_x64_Release_AllSubsets_Mono_RuntimeTests","BuildId":"218088","BuildNumber":"20230327.1","DefinitionId":"154"} | Schedule
</details>

**Comment by EgorBo** [comment]
Can't reproduce locally and the latest jitstress run seems to be clean?

**Comment by jakobbotsch** [comment]
> Can't reproduce locally and the latest jitstress run seems to be clean?

Yeah, I'm pretty sure I broke this with #88665 and fixed it with #88669... let's close it.
